src/main/java/yablunovskyi/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,55 @@
+package yablunovskyi.bookstore.controller;
+
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import yablunovskyi.bookstore.dto.BookDto;
+import yablunovskyi.bookstore.dto.CreateBookRequestDto;
+import yablunovskyi.bookstore.service.BookService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/api/books")
+public class BookController {
+    private final BookService bookService;
+    
+    @GetMapping
+    public List<BookDto> findAll() {
+        return bookService.findAll();
+    }
+    
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/{id}")
+    public BookDto findById(@PathVariable Long id) {
```

Comment:
you also can add @validated annotation on your controller class in order to make validation of @PathVariable, @RequestParam etc. (optional)
For example 
```suggestion
    public BookDto findById(@PathVariable @Positive Long id) {
```